docs/components/password/index.html

```diff
@@ -0,0 +1,143 @@
+---
+title: Password
+minver: 0.19.0
+also:
+  components/password/test.html: Testing - Password
+  elements/hx-password-control/index.html: <hx-password-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    Use {{page.title}} anytime a user must enter or update a password.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="basic-password-input">Password</h2>
+      {# TODO: add section description #}
+    </header>
+     <p>Build out the password entry interaction including:
+        Meeting or not meeting password requirements.  Consider a popover on focus that interactively displays met criteria.
+        Showing and hiding password text.
+    </p>
+
+    <div class="example" id="vue-passwordInputDemo" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <fieldset>
+            <legend class="beta-hxFieldName">Label Annotation</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkHasAsterisk"
+                type="checkbox"
+                v-model="hasAsterisk"
+              />
+              <label for="chkHasAsterisk">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+              <input
+                id="chkHasHelpText"
+                type="checkbox"
+                v-model="hasHelpText"
+              />
+              <label for="chkHasHelpText">
+                <hx-checkbox></hx-checkbox>
+                Help Text
+              </label>
+            </hx-checkbox-control>
+            <hx-checkbox-control>
+              <input
+                id="chkHasError"
+                type="checkbox"
+                v-model="hasError"
+              />
+              <label for="chkHasError">
+                <hx-checkbox></hx-checkbox>
+                Error Text
+              </label>
+            </hx-checkbox-control>
```

Comment:
1. Please move these into their own `<fieldset>`.
2. Toggle the demo `Help Text` and `Error Text`.
3. Always show these options in the legend.  No need to hide the inputs.

<img width="821" alt="Screen Shot 2020-03-23 at 7 27 25 PM" src="https://user-images.githubusercontent.com/10120600/77376190-b37d7e00-6d3d-11ea-891f-890a0696fe75.png">

Reply:
Changes have been done, please review.